app/models/api/v3/contextual_layer.rb

```diff
@@ -3,6 +3,8 @@ module V3
     class ContextualLayer < BaseModel
       belongs_to :context
 
+      has_one :carto_layer
```

Comment:
this should be a `has_many`, allowing to have different carto layers for different years